Spokestack/AppleSpeechRecognizer.swift

```diff
@@ -36,6 +39,10 @@ import Speech
     // MARK: NSObject implementation
     
     deinit {
+        if recognizingSemaphore.wait(timeout: .now() + self.configuration.semaphoreTimeout) == .timedOut {
+            self.context.dispatch { $0.failure(error: SpeechPipelineError.illegalState("Could not start recognizer, timed out waiting on another thread.")) }
```

Comment:
This is a long error message to be copy pasta'd everywhere.

Reply:
error message is pretty short, error handling is rather officious